packages/app/src/components/Button.tsx

```diff
@@ -2,24 +2,25 @@ import clx from "classnames";
 import React, { ReactNode } from "react";
 
 interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
-  variant?: "primary" | "secondary";
+  variant?: "green" | "red" | "unactive";
```

Comment:
I'd rather stay with states than colors. How about primary, secondary and danger (for red)

Reply:
Done